rbc/tests/test_omnisci_math.py

```diff
@@ -355,27 +356,22 @@ def test_numpy_function(omnisci, fn_name, signature, np_func):
             msg = str(msg).splitlines()[1]
             pytest.skip(msg)
 
-    if fn_name in ['spacing', 'nextafter']:
+    if fn_name in ['spacing']:
         pytest.skip(f'{fn_name}: FIXME')
 
-    if omnisci.has_cuda and fn_name in [
-            'arcsin', 'arccos', 'arctan', 'arctan2', 'hypot', 'sinh', 'cosh',
-            'tanh', 'arcsinh', 'arccosh', 'arctanh', 'expm1', 'exp2', 'log2',
-            'log1p', 'logaddexp2']:
-        # https://github.com/xnd-project/rbc/issues/59
-        pytest.skip(f'{fn_name}: crashes CUDA enabled omniscidb server'
-                    ' [rbc issue 59]')
-
-    if omnisci.has_cuda and fn_name in ['logaddexp']:
-        # https://github.com/xnd-project/rbc/issues/60
-        pytest.skip(f'{fn_name}: crashes CUDA enabled omniscidb server'
-                    ' [rbc issue 60]')
-
-    if omnisci.has_cuda and fn_name in ['lcm', 'ldexp']:
+    if omnisci.has_cuda and fn_name in ['lcm']:
```

Comment:
This check can be removed after #201 is merged

Reply:
#201 is merged.